### PR TITLE
Pausing RabbitMQ may not be working properly

### PR DESCRIPTION
**Pull request by ssOlexBaiko**

### CODE REVIEW

File: bin/quiesce-rabbitmq.sh [hunk 1]
```diff
@@ -182,7 +182,7 @@ function main()
     VARDIR="/etc/rabbitmq"
     [[ -n "$VARDIR" ]] || die "VARDIR env var is not set"
     [[ -d "$VARDIR" ]] || die "VARDIR=$VARDIR is not a directory"
-    export RABBITMQ_NODENAME="rabbit@localhost"
+    export RABBITMQ_NODENAME=$(rabbitmqctl status | grep -o '\brabbit@\w*')
```
Comment (vikt0rs) [inline bin/quiesce-rabbitmq.sh]: can we use `rabbit@$(hostname -s)` here?

Comment (glujan) [inline bin/quiesce-rabbitmq.sh]: Agreed

**Comment by glujan** [comment]
retest this

**Comment by ssOlexBaiko** [comment]
retest this

**Pull request merged**